shared/components/modals/OfferModal/AddOffer/AddOffer.js

```diff
@@ -119,13 +122,26 @@ export default class AddOffer extends Component {
       ? balance.plus(unconfirmedBalance)
       : balance
 
+    const finalBalance = BigNumber(currentBalance).minus(this.state.minimalestAmountForSell) > 0 ? BigNumber(currentBalance).minus(this.state.minimalestAmountForSell) : 0
```

Comment:
* currentBalance - уже является BigNumber, поэтому можно сразу использовать методы BigNumber
* Если используешь BigNumber для сравнения, делай это правильно: https://mikemcl.github.io/bignumber.js/#gt
* Если видишь, что в сравнении выходит дублирование выражения длинней его семантического названия, создай для него переменную с этим названием

```suggestion
    const balanceWithoutFee = currentBalance.minus(this.state.minimalestAmountForSell)
    const finalBalance = balanceWithoutFee.isGreaterThan(0) ? balanceWithoutFee : 0
```